### DISCUSSION TITLE
systemd

### DISCUSSION THREAD

**Discussion by zcatav** [SpiralLinux/SpiralLinux-project#53] [General]
Hi,
You are creator and maintainer of two fantastic GNU/Linux releases. 
Would you consider creating an alternative version like Devuan that doesn't contain systemd?
Thanks for your efforts.

**Comment by geckolinux** [comment]
Hi there, thanks for the question. It's actually closer than you might think to being possible since SpiralLinux is built with `live-build` , which also works for Devuan. So users that really want it could download one of the SpiralLinux build recipes from [here](https://github.com/SpiralLinux/SpiralLinux-project) and try to build it with `live-build` on a Devuan host system.

**Comment by zcatav** [comment]
Hi,
Thanks for information but this is beyond my abilities. I'm not an IT
person.

Zeki Çatav
zekicatav.com

17 Tem 2022 Paz 16:30 tarihinde GeckoLinux ***@***.***> şunu
yazdı:
